x-pack/legacy/plugins/ml/server/models/data_recognizer/modules/siem_auditbeat_ecs/ml/suspicious_login_activity_ecs.json

```diff
@@ -0,0 +1,36 @@
+{
+  "job_type": "anomaly_detector",
+  "description": "SIEM Auditbeat: Detect unusually high increases in authentication attempts (beta)",
+  "groups": ["siem"],
+  "analysis_config": {
+    "bucket_span": "15m",
+    "detectors": [
+      {
+        "detector_description": "high rate of authentication attempts",
+        "function": "high_non_zero_count",
+        "partition_field_name": "host.name"
+      }
+    ],
+    "influencers": [
+      "host.name",
+      "user.name",
+      "source.ip"
+    ]
+  },
+  "analysis_limits": {
+   "model_memory_limit": "256mb"
+  },
+  "data_description": {
+    "time_field": "@timestamp",
+    "time_format": "epoch_ms"
+  },
+  "custom_settings": {
+    "created_by": "ml-module-siem-auditbeat",
+    "custom_urls": [
+      {
+        "url_name": "IP Address Details",
+        "url_value": "siem#/network/ip/$source.ip$?_g=()&kqlQuery=(filterQuery:!n,queryLocation:network.details,type:details)&timerange=(global:(linkTo:!(timeline),timerange:(from:'$earliest$',kind:absolute,to:'$latest$')),timeline:(linkTo:!(global),timerange:(from:'$earliest$',kind:absolute,to:'$latest$')))"
```

Comment:
this custom URL does require that the ambiguity of `$source.ip$` (no or many influencers) be handled in the SIEM, being tracked here: https://github.com/elastic/ingest-dev/issues/564

do we release this with some kind of warning of these potential issues?

Reply:
Yes, the plan is to leave in the URLs which use influencers for now, with the plan of cleaning up no / many influencer value cases on the SIEM side in elastic/ingest-dev#564